TangemStaking/Services/StakingManager/CommonStakingManager.swift

```diff
@@ -82,12 +89,16 @@ extension CommonStakingManager: StakingManager {
             try await waitForLoadingCompletion()
             return try await estimateFee(action: action)
         case (.availableToStake, .stake), (.staked, .stake):
-            return try await provider.estimateStakeFee(
-                request: mapToActionGenericRequest(action: action)
+            return try await execute(
+                try await provider.estimateStakeFee(
+                    request: mapToActionGenericRequest(action: action)
+                )
```

Comment:
не крит, но вроде два раза `try await` можно не писать

```suggestion
            return try await execute(provider.estimateStakeFee(
                    request: mapToActionGenericRequest(action: action)
                )
```

Reply:
@fedorov-d посмотри плиз, уже отдельно потом в девелоп